src/test/java/teammates/test/cases/common/UrlTest.java

```diff
@@ -15,5 +17,56 @@ public void testTrimTrailingSlash(){
         assertEquals("abc.com", Url.trimTrailingSlash("abc.com/ "));
         assertEquals("abc.com", Url.trimTrailingSlash("abc.com"));
     }
+    
+    @Test
+    public void testToString() {
+        Url url = new Url("http://www.google.com", null);
+        assertEquals("", url.toString());
+        assertEquals("http://www.google.com", url.toAbsoluteString());
```

Comment:
A line comment to explain each test case? e.g.
`//typical base URL, null URL, empty string expected`